### PR TITLE
docs(setup) Updates to install Halyard and K8s and Docker-Registry provider setup

### PR DESCRIPTION
**Pull request by dorbin**

### CODE REVIEW

File: setup/install/halyard.md
Comment (lwander) [inline setup/install/halyard.md]: Let me fix this link before we merge...

File: setup/install/providers/docker-registry.md
Comment (lwander) [inline setup/install/providers/docker-registry.md]: I think this only throws a warning, so this is more of a soft-requirement

File: setup/install/providers/docker-registry.md
Comment (lwander) [inline setup/install/providers/docker-registry.md]: Maybe add ".. and you are responsible for refreshing it"

**Review by lwander — APPROVED** [review]

File: setup/install/providers/docker-registry.md
Comment (dorbin) [reply setup/install/providers/docker-registry.md]: Halyard throws the warning?

Comment (lwander) [reply setup/install/providers/docker-registry.md]: Yup, but it's non-blocking

**Review by avh121 — APPROVED** [review]

File: setup/install/index.md
Comment (avh121) [inline setup/install/index.md:32]: Certainly not something to address in this PR, but I think it's a little confusing that "configure everything else" and "productionize Spinnaker" are separate sections that seem to both be required to run Spinnaker in production.

File: setup/install/providers/docker-registry.md
Comment (avh121) [inline setup/install/providers/docker-registry.md]: s/reegistry/registry

File: setup/install/providers/kubernetes.md
Comment (avh121) [inline setup/install/providers/kubernetes.md]: s/setting/setting up/

**Review by avh121 — APPROVED** [review]

File: setup/install/index.md
Comment (ezimanyi) [inline setup/install/index.md:20]: Is the intention here that we'll be strongly pushing users to install to Kubernetes going forward?

File: setup/install/providers/kubernetes.md
Comment (ezimanyi) [inline setup/install/providers/kubernetes.md]: This link looks broken

**Review by ezimanyi — APPROVED** [review]

File: setup/install/index.md
Comment (dorbin) [reply setup/install/index.md:32]: Yeah, I was just noticing that.

File: setup/install/index.md
Comment (dorbin) [reply setup/install/index.md:20]: Yes

File: setup/install/index.md
Comment (dorbin) [reply setup/install/index.md:32]: @lwander , what if "Productionize Spinnaker" were moved under "Configure Everything Else"?

Comment (lwander) [reply setup/install/index.md:32]: I had it at the top-level since it feels like a large section, is immediately visible to anyone curious about Spinnaker & how it scales. It's also thematically a little different since it requires more understanding of Spinnaker than the other sections, and will likely require gathering & reading your own metrics/logs before deciding what to do.

Comment (dorbin) [reply setup/install/index.md:32]: Fair enough. We'll have to come up with some other way of dealing with the fact that "configure everything else" isn't really everything else.

Comment (avh121) [reply setup/install/index.md:32]: What if we changed it to something like "set up remaining services"  instead of "configure everything else"? I think that would make it a little clearer that we're dividing the docs into a first pass Spinnaker installation, thinking about things like metrics/security/etc, and finally a fully production-ready instance.

Comment (skim1420) [reply setup/install/index.md:32]: +1 to separate top-level section for productionize. We get enough inquiries in this realm, and it's useful toward emphasizing the battle-hardening we have. Also, generally I support the notion that Nora is bringing up, but this feels pretty non-controversial to solve.

File: setup/install/index.md
Comment (skim1420) [reply setup/install/index.md:20]: I don't know about "strongly pushing", however given a lack of particular preference from the user we want to highlight a primary path. This is strictly responsive to our installation option fan-out looking very large and overwhelming.

File: setup/install/halyard.md
Comment (skim1420) [inline setup/install/halyard.md]: What version of macOS has this been validated on? We should indicate, even if loosely. At least make clear it's not macos9 :)

**Review by skim1420 — APPROVED** [review]
LGTM after addressing my macOS version comment.

File: setup/install/halyard.md
Comment (dorbin) [reply setup/install/halyard.md]: @lwander ?

Comment (lwander) [reply setup/install/halyard.md]: High sierra

**Comment by lwander** [comment]
We never released the new halyard version for macos -- this was merged too early.

**Pull request merged**
